doc/devel/contribute.rst

```diff
@@ -161,6 +161,29 @@ please reach out on the :ref:`contributor_incubator`
 
 .. _`open an issue`: https://github.com/matplotlib/matplotlib/issues/new?assignees=&labels=Documentation&projects=&template=documentation.yml&title=%5BDoc%5D%3A+
 
+.. _contribute_triage:
+
+Triage
+------
+The `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_ is our
+centralized location for feature requests, bug reports,
+tracking major projects and discussing priorities. It takes a lot of
+time and effort to keep oragnized, so curating our issue tracker is a valuable
+contribution. Some examples are:
+
+* labeling issues and pull requests
+* verifying bug reports
+* debugging and resolving issues
+* linking to related issues, discussion, and external work
+
+Helping out with triage is discussed in detail in :ref:`bug_triaging`.
+
+Triage help is a valuable contribution because it takes a lot of time and
+judgement to effectively curate and resolve issues; therefore we very much
+appreciate thoughtful feedback to issue authors. Please do not verbatim post
+the output of a generative AI tool (such as chat GPT) as a response to a bug
+report as we expect that the issue author has either already tried that or has a
+reason for not doing so.
```

Comment:
This is a way of working aspect of triaging. It belongs under the above linked `bug triaging` docs, not here. - My idea of "Contributing guide > Getting started" is that it should give a high-level idea of the kinds contribution and direct interested people to the relevant parts of the documentation. Specific information on how (not) to do things should be in those parts of the documentation e.g. `bug triaging`, not on the overview page.

Reply:
My first impulse was: This belongs in https://matplotlib.org/devdocs/devel/triage.html. But considering the definition of triage actions on that site, replying with a solution is beyond triaging. Triaging only makes sure the issue is well laid out and sorted. It does not solve it (except for invalid or duplicate issues). Therefore, I don't see that people will respond with AI as part of triaging. To make life simple here, I claim it's out of scope for the PR and we can leave that section out.

You could add something like "Proposing solutions" to the https://matplotlib.org/devdocs/devel/triage.html. It should be made clear that
- it's optional (a triager may add it if they already exactly know how to solve this)
- proposals / recommendations should be well considered and well laid out. If you are not sure, better leave the response to somebody else. This also includes auto-generated AI responses are not acceptable.